hapi/hedera-protobufs/services/response_code.proto

```diff
@@ -1637,5 +1637,11 @@ enum ResponseCodeEnum {
     * we cannot add it on update.
     */
     FEE_SCHEDULE_KEY_CANNOT_BE_UPDATED = 377;
-
+  
+    /**
+    * If the topic's custom fees are updated the topic SHOULD have a
+    * fee schedule key
```

Comment:
Those * are not aligned correctly. You will have to update before merging to develop.

Reply:
Yeah, when we merge this PR to `hip-991-topic-fees` and we finalize it we can align the proto messages 👍